src/main/java/fi/aalto/cs/apluscourses/ui/module/ModulesView.java

```diff
@@ -49,6 +50,7 @@ public ModulesView() {
     // We use class name as a unique key for the property.
     basePanel.putClientProperty(ModulesView.class.getName(), this);
     cl = (CardLayout) cardPanel.getLayout();
+    moduleListView.setBackground(new Tree().getBackground());
```

Comment:
A bit unusual... but ok, I guess.

I looked at the implementation of `Tree` and it delegates to `com.intellij.util.ui.UIUtil::getTreeBackground`. Calling that method here would kinda make more sense but because `UIUtil` is not part of the "openapi" it's not that good idea to depend on it.